src/ibbtc_fee_collector.py

```diff
@@ -70,6 +70,9 @@ def get_outstanding_fees(self) -> Decimal:
         return Decimal(raw_fees / 10 ** 18)
 
     def __is_profitable(self, fees: Decimal) -> bool:
+        if fees == 0:
```

Comment:
Can't compare Decimal to integer properly. Suggest doing `if fees == Decimal(0):`